api/v1alpha3/machine_types.go

```diff
@@ -37,6 +37,16 @@ const (
 
 	// MachineDeploymentLabelName is the label set on machines if they're controlled by MachineDeployment
 	MachineDeploymentLabelName = "cluster.x-k8s.io/deployment-name"
+
+	// PreDrainDeleteHookAnnotionPrefix annotation specifies the prefix we
+	// search each annotation for during the pre-drain.delete lifecycle hook
+	// to pause reconciliation.
```

Comment:
Maybe a compromise:
```suggestion
	// to pause reconciliation of deletion.
```
Since the definition of these hooks should not impact reconciliation of resources that are not pending deletion